docs/components/integration-framework/connectors/out-of-the-box-connectors/googledrive.md

```diff
@@ -0,0 +1,200 @@
+3. In the **Operation Details** section, set the field **Folder name** as the desired name of a folder you wish to create. For example, `MyNewFolder`. Alternatively, you could use a FEEL expression.
+4. _(optional)_ In the **Operation Details** section, set the field **Parent folder ID** to the desired parent, inside which a new folder will be created. Please, keep in mind, that if not specified, a new folder will be created in the Google Drive root folder of a user, who owns the OAuth token.
+5. _(optional)_ In the **Operation Details** section, set the field **Additional properties or metadata**, set Google Drive compatible properties or metadata. This property requires FEEL input. Check the appendix for known values and limitations.
+
+## Make your Google Drive connector for creating a file from template executable
+
+To make the **Google Drive connector** executable, fill out the mandatory fields highlighted in red in the properties panel:
+
+![Google Drive connector new file from template example](../img/connectors-googledrive-new-file-filled.png)
+
+1. In the **Authentication** section, set the field Google Drive Bearer Token to your OAuth token, for example, `secrets.GDRIVE_OAUTH_TOKEN`.
+2. In the **Select Operation** section, set the field value **Operation Type** as **Create File from template**.
+3. In the **Operation Details** section, set the field **File name** as the desired name of a folder you wish to create. You can use FEEL expressions here.
+4. In the **Operation Details** section, set the field **Template ID** of the desired template.
+5. _(optional)_ In the **Operation Details** section, set the field **Parent folder ID** to the desired parent, inside which a new file will be created. Please, keep in mind, that if not specified, a new folder will be created in the Google Drive root folder of a user, who owns the OAuth token.
+6. In the **Operation Details** section, set the field **Template variables** as desired variables that will be applied to the template. The template variables a compatible with the Google Docs Requests API. This property requires FEEL input.
+7. _(optional)_ In the **Operation Details** section, set the field **Additional properties or metadata**, set Google Drive compatible properties or metadata. This property requires FEEL input. Check the appendix for known values and limitations.
+
+## Google Drive connector response
+
+The **Google Drive connector** exposes Google Drive API response as a local variable called response.
+The following fields are available in the response variable:
+`googleDriveResourceId` - ID of a newly created resource
+`googleDriveResourceUrl` - human-readable URL of a newly created resource
+
+You can use an Output Mapping to map the response:
+
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables. For example:
+
+```
```

Comment:
I don't get the example. The expression you put would map static fields into variables right?

So after execution the variable `googleDriveResourceId` would exist in my process with the value `1KMG4S_18kLNZa7ZCARtGIh8DmIoXXXXXXXXX`. Why would I do that?

Would a more appropriate example not be something like this:

```
{
  "newFolderID": response.googleDriveResourceId
}
```

Reply:
Or do I miss something?

Reply:
Yes, correct. Updated.